src/extension_utils/utils.ts

```diff
@@ -164,45 +168,30 @@ export function generateCPXConfig(): void {
     mkdirRecursivelySync(path.dirname(cpxConfigFilePath));
     fs.writeFileSync(cpxConfigFilePath, JSON.stringify(cpxJson, null, 4));
 }
-export const checkPythonDependency = async () => {
-    const dependencyChecker: DependencyChecker = new DependencyChecker();
-    const result = await dependencyChecker.checkDependency(
-        CONSTANTS.DEPENDENCY_CHECKER.PYTHON
-    );
-    return result.payload;
-};
-
-export const checkPipDependency = async () => {
-    const dependencyChecker: DependencyChecker = new DependencyChecker();
-    const result = await dependencyChecker.checkDependency(
-        CONSTANTS.DEPENDENCY_CHECKER.PIP3
-    );
-    return result.payload;
-};
 
-export const setPythonExectuableName = async () => {
-    // Find our what command is the PATH for python
-    let executableName: string = "";
-    const dependencyCheck = await checkPythonDependency();
-    if (dependencyCheck.installed) {
-        executableName = dependencyCheck.dependency;
-    } else {
+export const checkForPip = async (pythonExecutableName: string) => {
```

Comment:
nit: this is all nit suggestions, so don't worry all much about it. I would use the "check" keyword for methods that validate something (throw an exception or show up an error window).

We can be more descriptive for methods that return a boolean (such as this one), e.g. `IsPipInstalled()`

Reply:
ah ok! I'll make some changes for that